src/main/java/org/eclipsefoundation/marketplace/model/Error.java

```diff
@@ -0,0 +1,88 @@
+/*
+ * Copyright (C) 2019 Eclipse Foundation and others.
+ * 
+ * This program and the accompanying materials are made
+ * available under the terms of the Eclipse Public License 2.0
+ * which is available at https://www.eclipse.org/legal/epl-2.0/
+ * 
+ * SPDX-License-Identifier: EPL-2.0
+*/
+package org.eclipsefoundation.marketplace.model;
+
+import javax.ws.rs.core.Response;
+import javax.ws.rs.core.Response.Status;
+
+public class Error {
```

Comment:
I don't think this class is required to be mutable. Once the fields are set in the constructor, I don't see the use case for modifying them. I'd suggest to remove all setters and and set all fields as final